README.md

```diff
@@ -36,13 +36,13 @@ Please make sure to assign a valid signing configuration to your release build t
 
 ## Usage
 
-Add it to your buildscript dependencies:
+Add it to your buildscript dependencies(top-level build.gradle file):
 
 ```groovy
 buildscript {
 
     repositories {
-        mavenCentral()
+        mavenCentral() // or maybe jcenter() depends your project's configuration
```

Comment:
Please revert this line. `jcenter()` is a super-set of `mavenCentral()`, but as we publish to `mavenCentral()` this should be enough.